index copy.html

```diff
@@ -0,0 +1,298 @@
+            <div class="middle-image"></div>
+            <div class="side-image"></div>
+            <div class="image-geometry-2"></div>
+            <div class="image-geometry-5"></div>
+            <div class="image-geometry-4"></div>
+            <div class="image-geometry-3"></div>
+            <div class="rectangle-25-1"></div>
+            <div class="image-teal"></div>
+            <div class="projects-dots"></div>
+            <div class="name-dots"></div>
+        </div>
+        <div>
+            <h2 class="projects" id="site-project">Projects</h2>
+            <div class="computer-image"></div>
+            <div class="large-computer-image"></div>
+            <div class="project-section">
+                <h2>Project name goes here</h2>
+            </div>
+            <div class="grid-container">
+                <div class="html-css-rectangle">
+                    <p>HTML/CSS</p>
+                </div>
+                <div class="ruby-rails-rectangle">
+                    <p>Ruby on Rails</p>
+                </div>
+                <div class="javascript-rectangle">
+                    <p>Javascript</p>
+                </div>
+                <button data-modal-target="#modal" type="button" class="see-project">See this project</button>
+                <div class="arrow"></div>
+            </div>
+            <div class="image-geometry-4"></div>
+            <div class="macbook-image"></div>
+            <div class="project-section-1">
+                <h2>Project name goes here</h2>
+            </div>
+            <div class="grid-container-1">
+                <div class="html-css-rectangle">
+                    <p>HTML/CSS</p>
+                </div>
+                <div class="ruby-rails-rectangle">
+                    <p>Ruby on Rails</p>
+                </div>
+                <div class="javascript-rectangle">
+                    <p>Javascript</p>
+                </div>
+                <button data-modal-target="#modal" type="button" class="see-project">See this project</button>
+                <div class="arrow"></div>
+            </div>
+            <div class="mac-iphone-image"></div>
+            <div class="project-section-2">
+                <h2>Project name goes here</h2>
+            </div>
+            <div class="grid-container-2">
+                <div class="html-css-rectangle">
+                    <p>HTML/CSS</p>
+                </div>
+                <div class="ruby-rails-rectangle">
+                    <p>Ruby on Rails</p>
+                </div>
+                <div class="javascript-rectangle">
+                    <p>Javascript</p>
+                </div>
+                <button type="button" class="see-project">See this project</button>
+                <div class="arrow"></div>
+            </div>
+            <div class="mac-right-image"></div>
+            <div class="project-section-3">
```

Comment:
- [ ] Kindly make sure all `see project` buttons rendered a modal for that button, at the moment, only one button is responding.
- [ ] Also make sure the project details of each card are visible when it's hovered.